src/Culling/babylon.boundingBox.ts

```diff
@@ -64,52 +57,82 @@
             this.reConstruct(min, max);
         }
 
+        /**
+         * Gets the 8 vectors representing the bounding box in local space
+         */
+        public get vectors(): Vector3[] {

```

Comment:
This will also be a HUGE perf hit for Solidparticlesystem.
I definitely think we should have a specialized version of the BoundingBox (like BoundingBoxLite) that will be used by Babylonjs based on a general flag

Reply:
actually this getter would ideally be declared deprecated and not be used by anything within the engine.

Reply:
We can't deprecate public API as there may be a lot of users out there